i-slide.js

```diff
@@ -127,6 +120,13 @@ class ISlide extends HTMLElement {
 
     try {
       const resp = await fetch(docUrl);
+      if (resp.status !== 200) {
+        console.error(
+          'Could not fetch slide deck', docUrl,
+          'HTTP status code received is', resp.status);
+        cache[docUrl] = { type: 'error', status: resp.status };
```

Comment:
status isn't used anywhere else, is this on purpose?

Reply:
Sorta. I thought it could be useful later on if we wanted to render a specific message e.g. when an internal server error gets received. But then we can easily add the `status` property when we actually consider that. I'll drop it for now.